homeassistant/components/calendar/google.py

```diff
@@ -14,6 +14,8 @@
     GoogleCalendarService)
 from homeassistant.util import Throttle, dt
 
+REQUIREMENTS = ["httplib2==0.10.3"]
```

Comment:
Don't add this. Requirements are installed as part of the parent component.

Reply:
I don't understand where I'm supposed to add it. In the `__init__.py` for the `calendar` component?

Reply:
If I don't add it, the tox unit testing will fail since the `gen_requirements_all.py` script seems to only add test requirements from the `TEST_REQUIREMENTS` tuple if they're actually listed somewhere in a module.

Reply:
Unless you'd rather I edit `requirements_test.txt` instead?

Reply:
The requirement that depends on this package needs to be whitelisted.

Reply:
Alternatively, you can use the `MockDependency` decorator from `tests.common`.